nucypher/blockchain/eth/policies.py

```diff
@@ -175,23 +158,27 @@ def make_arrangements(self, network_middleware: RestMiddleware,
         selected_addresses = set()
         try:  # Sample by reading from the Blockchain
             actual_sample_quantity = math.ceil(target_sample_quantity * ADDITIONAL_URSULAS)
-            sampled_addresses = self.alice.recruit(quantity=actual_sample_quantity)
+            duration = int(calculate_period_duration(expiration))
```

Comment:
Here is an example of the time/period conversion.

Reply:
Aye.  :+1: